tests/e2e/k8s-keystone-auth/test.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
```

Comment:
Just a suggestion ,
may be filename with more specific as test-auth-policy-role-mapping.sh , so that any future tests also can be added accordingly?

Reply:
Do you mean using different script files for different functional tests?

Reply:
@ramineni please let me know what do you think?

Reply:
Done